librz/arch/isa/msp430/msp430_il.c

```diff
@@ -0,0 +1,199 @@
+// SPDX-FileCopyrightText: 2023 Mostafa Mahmoud <ubermenchun@gmail.com>
+// SPDX-License-Identifier: LPGL-3.0-only
+
+#include "msp430_il.h"
```

Comment:
Please include them with `<msp430/msp430_il.h>`

Reply:
Do you mean the copyright notices? I did write them in the `<msp430/msp430_il.h>`. Or did you mean that I should remove them from the `.c` and rely on the `#include` to copy them by itself?

Reply:
```suggestion
#include <msp430/msp430_il.h>
```

Sorry, missed this one. This is the suggestion I had.